include/RAJA/config.hxx.in

```diff
@@ -154,102 +154,106 @@ const int COHERENCE_BLOCK_SIZE = @RAJA_COHERENCE_BLOCK_SIZE@;
 //
 //     RAJA_INLINE - macro to enforce method inlining
 //
-//     RAJA_ALIGN_DATA(<variable>) - macro to express alignment of data, 
-//                              loop bounds, etc.
+//     RAJA_SIMD - macro to apply compiler-specific pragma that encourages
+//                 compiler to SIMD vectorize loop even if it may not
+//                 normally
+//
+//     RAJA_SIMDFORCE - macro to apply compiler-specific pragma that forces
```

Comment:
The C++ proposal for parallel standard algorithms uses 'seq', 'par', and 'vec'. Seq executes in strictly sequential order. Par executes (optionally) in parallel in any order defined by the implementation choice. Vec executes similarly to par with additional constraints on exceptions and synchronization. For example, vec could use #pragma omp parallel for simd internally.

This doesn't exactly map onto what we are trying to do since there is no concept of "forcing" parallel execution (e.g., SIMD or multithreading). Should we try to be closer to standard concepts/names?